src/pages/notification/index.js

```diff
@@ -0,0 +1,59 @@
+import "./notification.css";
+import { useState, useEffect } from "react";
+import Header from "../../components/admin-header/Header";
+import Sidebar from "../../components/admin-sidebar/SideBar";
+import { FaBell } from "react-icons/fa";
+import BreadCrumb from "../../components/BreadCrumb/BreadCrumb";
+import { connect } from "react-redux";
+import NotificationComponent from "../../components/notification/notificationComponent";
```

Comment:
This should be `import NotificationComponent from "../../components/notification/NotificationComponent";`